src/components/map/rent-popup.tsx

```diff
@@ -48,12 +50,14 @@ const RentControls: React.FC<RentControlsProps> = ({
     [pinInput],
   );
 
-  const { map, MAP } = useContext(LanguageContext);
+  const { map, BUCHUNGEN } = useContext(LanguageContext);
+  const [stations] = useStations();
```

Comment:
Maybe we should pass in the stations from above via prop instead of loading them here via hook. This will load them twice (once when loading the map and once when opening the popup).

Reply:
Actually, the `RentPopup` already receives the stations as prop!

Reply:
See fad5ec0660947a692790f284be62540995b15395.
Additionally, `isCurrentStationBooked` has been renamed to match the `openedStation` prop (and its definition has been simplified).